source/includes/config-repo/_022-pipeline-export.md.erb

```diff
@@ -0,0 +1,132 @@
+## Pipeline export
+
+This message is a request to the plugin to convert a pipeline configuration into the plugin's native language. It is
```

Comment:
native format, rather than native language?

Reply:
I debated between the two - I chose to use "language" because we do interpret it semantically (and in the groovy case it is a bona fide language). But "format" does seem more natural, though it is technically less accurate.